docs/src/get-started.md

```diff
@@ -0,0 +1,191 @@
+p = Mamba.plot(c3)
+Mamba.draw(p, fmt=:pdf, filename="gdemo-plot.pdf")
+```
+
+The arguments for each sampler are
+
+* SMC: number of particles
+* PG: number of particles, number of iterations
+* HMC: number of samples, leapfrog step size, leapfrog step numbers
+* Gibbs: number of samples, component sampler 1, component sampler 2, ...
+* HMCDA: number of samples, total leapfrog length, target accept ratio
+* NUTS: number of samples, target accept ratio
+
+For detailed information please check Turing.jl's [APIs](https://github.com/yebai/Turing.jl/wiki/APIs).
+
+### Modelling syntax explained
+
+Models are wrapped by `@model` with a normal function definition syntax, i.e.
+
+```julia
+@model model_name(arg_1, arg_2) = begin
+  ...
+end
+```
+
+This syntax defines a model which can take data as input to generate a posterior evaluator. The data can be provided either using the same function signature defined, or by using a dictionary containing each argument and its value as pairs, i.e.
+
+```julia
+model_func = model_name(1, 2)
+model_func = model_name(Dict(:arg_1=>1, :arg_2=>2)
+```
+
+This posterior evaluator can then be called by a sampler to run inference, i.e.
+
+```julia
+chn = sample(model_func, HMC(..)) # do inference by sampling using HMC
+```
+
+The return chain contains samples of the variables in the model, one can use them do inference, e.g.
+
+```julia
+var_1 = mean(chn[:var_1]) # taking the mean of a variable named var_1
+```
+
+Note that the key should be a symbol. For this reason, in case of fetching `x[1]` one need to do `chn[Symbol(:x[1])`. Turing.jl provides a macro to work around this expression `chn[sym"x[1]"]`.
+
+## Beyond basics
+
+### Composition sampling using Gibbs
+
+Turing.jl provides a Gibbs interface to combine different samplers. For example, one can combine a HMC sampler with a PG sampler to run inference for different parameters in a single model as below.
+
+```julia
+@model simple_choice(xs) = begin
+  p ~ Beta(2, 2)
+  z ~ Categorical(p)
+  for x = xs
+    if z == 1
+      x ~ Normal(0, 1)
+    else
+      x ~ Normal(2, 1)
+    end
+  end
+end
+
+simple_choice_f = simple_choice([1.5, 2.0, 0.3])
+
+chn = sample(simple_choice_f, Gibbs(1000, HMC(1,0.2,3,:p), PG(20,1,:z))
+```
+
+For details of composition sampling in Turing.jl, please check the corresponding [paper](http://xuk.ai/assets/aistats2018-turing.pdf).
+
+### Working with Mamba.jl
+
+Turing.jl wraps its samples using `Mamba.Chain` so that all the functions working for `Mamba.Chain` can be re-used in Turing.jl. Two typical functions are `Mamba.describe` and `Mamba.plot`, which can be used as follow for an obtained chain `chn`.
+
+```julia
+using Mamba: describe, plot
+
+describe(chn) # gives statistics of the samples
+plot(chn) # lots statistics of the samples
+```
+
+There are a plenty of functions which are useful in Mamaba.jl, e.g. those for convergence diagnostics at [here](http://mambajl.readthedocs.io/en/latest/tutorial.html#convergence-diagnostics).
```

Comment:
- `Mamaba` -> `Mamba`
- remove `at`

Reply:
It occurs to me that we don't even use Mamba anymore -- is that correct? Maybe this all needs to be transitioned to MCMCChain.

Reply:
Yes, `MCMCChain` supports the diagnostics functions from Mamba. We can refer to the README of MCMCChain as the functions are documented there.